src/menu.js

```diff
@@ -105,6 +105,12 @@ exports.get = function(props = {}) {
           label: i18n.t('menu.file', '&Preferences…'),
           accelerator: 'CmdOrCtrl+,',
           click: () => sabaki.openDrawer('preferences')
+        },
+        {type: 'separator'},
+        {
+          label: i18n.t('menu.file', '&Quit'),
+          accelerator: 'Alt+F4',
```

Comment:
We can't assign <kbd>Alt+F4</kbd> to quit, because on Windows <kbd>Alt+F4</kbd> is only supposed to close the current window not all windows of the application. <kbd>CmdOrCtrl+Q</kbd> is more appropriate.

Also, on macOS, the "Quit" menu item exists in [another menu already](https://github.com/SabakiHQ/Sabaki/blob/47a2693b15c6efaeed46a3e522cf6183dfbc0e68/src/menu.js#L898), so we need to hide these in macOS.

Reply:
I have changed the accelerator and hid the menu option on MacOS.